source/_patterns/00-base/_variables.scss

```diff
@@ -0,0 +1,37 @@
+///
+/// Global VARIABLES and HELPERS
+///
+/// Place global config here like Bootstrap's _variables.scss.
+/// This kind of config is often required prior to importing any libraries (ie Bootstrap Sass), and
+/// does not produce any CSS, just variables and maps.
+///
+/// All .scss files NEED TO INCLUDE this file as the top import to ensure consistent settings.
+/// Particle does not glob Sass! Because there is no printing CSS output in this file, webpack will
+/// safely squash any duplication.
+///
+/// @example
+/// ```scss
+/// @import '../../00-base/variables';
+/// ```
+/// This guarantees every component is styled with the same presets.
+///
+
+// Custom non-printing tools
+@import 'non-printing/functions';
+@import 'non-printing/colors';
+@import 'non-printing/mixins';
+
+// (Other non-printing .scss), ie font families, spacing, etc
+$spacing: (
+  s: 5px,
+  m: 25px,
+  l: 50px,
+);
+
```

Comment:
Comment right here that this is where we can override bootstrap variables (that aren't just overriden in a component later on).